certbot/main.py

```diff
@@ -5,6 +5,7 @@
 import functools
 import logging.handlers
 import sys
+import warnings
```

Comment:
Lint is angry about this unused import but this otherwise LGTM! Thanks for getting this PR ready.